test/visibility/import/rename/same_name_conflict2.chpl

```diff
@@ -0,0 +1,13 @@
+module A {
+  var x = 10;
+}
+module C {
+  var x = 15;
+}
+module D {
+  import C as A;  // implies A.x is 15
+  import A; // imports the renamed A from the previous import
```

Comment:
I'd like to see a test that does this kind of thing but at this point does something like `import A.submodule` or `import A.x`